src/index.html

```diff
@@ -11,7 +11,194 @@
+          <input 
+            type="date" 
+            id="birth" 
+            name="birth"
+          >
+        </div>
+
+        <div class="inputs">
+          <label for="agreement">I accept the terms of the agreement:</label>
+          <input 
+            type="checkbox" 
+            id="agreement" 
+            name="agreement"
+            required
+          >
+        </div>
+
+      </fieldset>
+        
+      <fieldset>
+        <legend>Registration</legend>
+
+
+        <div>
+          <label for="email">E-mail:</label>
+          <input 
+            type="email" 
+            id="email" 
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+          >
+        </div>
+
+
+        <div class="inputs">
+          <label for="password">Password:</label>
+          <input 
+            type="password" 
+            id="password" 
+            name="password"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+
+        <div>
+          <span>Do you love cats?</span>
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="yes"
+            >
+            Yes
+          </label>
+          
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+
+        <div class="inputs">
+          <label for="color">What is your favorite color?</label>
+          <input 
+            type="color" 
+            id="color" 
+            name="color"
+          >
+        </div>
+
+
+        <div class="inputs">
+          <label for="time">What time do you go to bed?</label>
+          <input 
+            type="time" 
+            id="time" 
+            name="time"
+          >
+        </div>
+
+
+        <div class="inputs">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select 
+            name="cars" 
+            id="cars" 
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="inputs">
+          <label for="rating">How do you rate our work?</label>
+          <input 
+            type="range" 
+            name="rating" 
+            id="rating"
+            value="0"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+
+
+        <div>
+          <label for="comments">Comments:</label>
+          <textarea 
+            name="comments" 
+            id="comments"
+            maxlength="100"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent 
<img width="299" alt="image" src="https://user-images.githubusercontent.com/53611910/216774916-8367fc5d-3eb6-4d6e-8c15-cde87cfe0be6.png">